main.nf

```diff
@@ -219,10 +232,11 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
  */
 process star_fusion {
     tag "$name"
+    conda ''
```

Comment:
Is this useful?

Reply:
Ah, I must have forgotten about that. I was testing something with conda. I will remove it.